testJITServer_1 testServerGoesDown AssertionError

https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_sanity.functional_x86-64_linux_xl_Nightly/290
testJITServer_1
```
xception in thread "main" java.lang.NullPointerException
	at sun.misc.URLClassPath$FileLoader.getResource(URLClassPath.java)
	at sun.misc.URLClassPath.getResource(URLClassPath.java:345)
	at java.net.URLClassLoader$ClassFinder.run(URLClassLoader.java:1084)
	at java.security.AccessController.doPrivileged(AccessController.java:770)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:589)
	at java.lang.ClassLoader.loadClassHelper(ClassLoader.java:953)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:898)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:881)
	at org.testng.reporters.TestHTMLReporter.generateLog(TestHTMLReporter.java:260)
	at org.testng.reporters.TestHTMLReporter.onFinish(TestHTMLReporter.java:40)
	at org.testng.TestRunner.fireEvent(TestRunner.java:772)
	at org.testng.TestRunner.afterRun(TestRunner.java:741)
	at org.testng.TestRunner.run(TestRunner.java:509)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.TestNG.privateMain(TestNG.java:1354)
	at org.testng.TestNG.main(TestNG.java:1323)
```
```
FAILED: testServerGoesDown
java.lang.AssertionError: 
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at org.testng.AssertJUnit.fail(AssertJUnit.java:66)
	at jit.test.jitserver.JITServerTest.destroyAndCheckProcess(JITServerTest.java:122)
	at jit.test.jitserver.JITServerTest.testServerGoesDown(JITServerTest.java:208)
```

--- a/test/functional/JIT_Test/playlist.xml
+++ b/test/functional/JIT_Test/playlist.xml
@@ -523,6 +523,7 @@
 		echo $(Q)$(TEST_JDK_BIN)$(D)jitserver doesn't exist; assuming this JDK does not support JITServer and trivially passing the test.$(Q); \
 	fi; \
 	$(TEST_STATUS)</command>
+		<disabled>https://github.com/eclipse/openj9/issues/8806</disabled>
 		<platformRequirements>os.linux,arch.x86,bits.64</platformRequirements>
 		<levels>
 			<level>sanity</level>

